### ISSUE TITLE
ide-python starts rogue bash process

### ISSUE THREAD

**Issue by pashadia** [lgeiger/ide-python#20]
Whenever I start atom (or whenever I enable ide-python in settings), there's a bash process that gets spawned. It clogs the CPU and it cannot be ended with SIGTERM, only SIGKILL.

The command, as shown in `ps` is `/bin/bash -ilc env; exit`

Other info:

```
$uname -a
Linux kramer-VirtualBox 4.10.0-35-generic #39-Ubuntu SMP Wed Sep 13 07:46:59 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
$atom --version
Atom    : 1.20.1
Electron: 1.6.9
Chrome  : 56.0.2924.87
Node    : 7.4.0
```

**Comment by lgeiger** [comment]
You might want to try Atom 1.21.0-beta2

In the [blog post](http://blog.atom.io/2017/09/12/announcing-atom-ide.html) the Atom team mentions:
>We strongly recommended you use **Atom Beta 1.21** as it includes the necessary file monitoring and process control to ensure the underlying language servers are running properly.

**Comment by culhatsker** [comment]
I have the same issue with Atom 1.21.0-beta2 and Atom 1.21.0 (not beta)
I have Ubuntu 16.04 LTS on my machine.
Looks like it hangs on https://github.com/lgeiger/ide-python/blob/master/lib/main.js#L32
I commented out this line (and line 35) and everything worked for me.
I'm not sure if this works for other platforms.

**Comment by renanvieira** [comment]
@culhatsker solution worked for me. I'm using Mint 16.

**Comment by lgeiger** [comment]
Thanks a lot for investigating!

`shell-env` was added to fix #4 (this may be only a problem on macOS though).

I think we should see if there's a better solution for this.

**Comment by lgeiger** [comment]
Fixed in `ide-python@0.2.2`

Thanks @kbrose for helping debug.
